### DISCUSSION TITLE
Which HTTP client (coroutine) recommended to complement Workerman speed?

### DISCUSSION THREAD

**Discussion by cayolblake** [walkor/Workerman#786] [General]
Hello,

I wonder what's the most recommended HTTP client (coroutine) that complements the speed of Workerman and not waste any resources or introduce overhead.

@walkor would appreciate your input here buddy!

**Comment by walkor** [comment]
https://github.com/walkor/http-client

**Comment by cayolblake** [comment]
Many many thanks 😊
